src/pydicom/dataset.py

```diff
@@ -2656,6 +2660,56 @@ def ensure_file_meta(self) -> None:
         if not hasattr(self, "file_meta"):
             self.file_meta = FileMetaDataset()
 
+    def validate(self, _init_value: _DatasetType) -> None:
```

Comment:
Why is `_init_value` required?

Reply:
Because `FileMetaDataset` inherits from `Dataset` and has its own `validate` function, `mypy` will complain about non-matching signatures.

Reply:
Oh, I missed that somehow. I'm not really sure why `FileMetaDataset.validate()` has been implemented like that... honestly I would've though it'd be better to just make it a standard class method with no args.